components/SchedulePage.js

```diff
@@ -0,0 +1,49 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'styled-components';
+
+import {HeaderImage, HeaderImageContainer, MainSection, Main} from './DefaultPage';
+import ContentBlock from './ContentBlock';
+import Footer from '../components/Footer.js';
+import Page from '../components/Page.js';
+import Schedule from './Schedule';
+
+const Section = styled.div`
+    margin: 0 auto;
+    width: 90%;
+`;
+
+export default class SchedulePage extends React.PureComponent {
+    static propTypes = {
+        headerImage: PropTypes.object.isRequired,
+        title: PropTypes.string.isRequired,
+        title2: PropTypes.string.isRequired,
+        seo: PropTypes.object.isRequired,
+        contentBlocks: PropTypes.array.isRequired,
```

Comment:
Is this just for the eventual case that we want to add some text on the page?

Reply:
yes